### PR TITLE
Release v0.1.1 to CRAN 

### LINKED ISSUE #283: Release epichains 0.1.1
First release:

* [x] `usethis::use_cran_comments()`
* [x] Update (aspirational) install instructions in README
* [x] Proofread `Title:` and `Description:`
* [x] Check that all exported functions have `@return` and `@examples`
* [x] Check that `Authors@R:` includes a copyright holder (role 'cph')
* [x] Check [licensing of included files](https://r-pkgs.org/license.html#sec-code-you-bundle)
* [x] Review <https://github.com/DavisVaughan/extrachecks>

Prepare for release:

* [x] `git pull`
* [x] `urlchecker::url_check()`
* [x] `devtools::build_readme()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [ ] Accepted :tada:
* [ ] Merge this PR into `main`
* [ ] `usethis::use_github_release()`
* [ ] `usethis::use_dev_version(push = TRUE)` (through separate PR)

### PR DESCRIPTION
**Pull request by jamesmbaazam**
This PR closes #283 by addressing the checklist in preparation for the CRAN submission. This PR will be merged into `main` as it will be the version bundled for the CRAN release.

### CODE REVIEW

File: README.md
Comment (sbfnk) [inline README.md:48]:
```suggestion
install.packages("epichains")
```

File: cran-comments.md
Comment (sbfnk) [inline cran-comments.md:3]: say what the note is and why it needs to be there?

**Review by sbfnk — APPROVED** [review]

**Comment by jamesmbaazam** [comment]
Dear James M. Azam
Someone has submitted the package epichains to CRAN.
You are receiving this email to confirm the submission as the maintainer of
this package.
To confirm the submission to CRAN, follow or copy & paste the following
link into your browser:

[https://xmpalantir.wu.ac.at/cransubmit/conf_mail.php?code=a961cc850d1bfa669557f748eb7a23a7](https://url.uk.m.mimecastprotect.com/s/60vDCj814C5Qlv7fYIWFmG4a1?domain=xmpalantir.wu.ac.at)

If you did not submit the package or do not wish for it to be submitted to
CRAN, simply ignore this email

Submission Information:
Submitter: James M. Azam <james.azam@lshtm.ac.uk>
Package: epichains
Version: 0.1.1
Title: Simulating and Analysing Transmission Chain Statistics Using
Branching Process Models
Author(s): James M. Azam [aut, cre, cph]
(<[https://orcid.org/0000-0001-5782-7330](https://url.uk.m.mimecastprotect.com/s/qwnGCk714TWzYpMCyS9FGNAnQ?domain=orcid.org)>), Sebastian Funk [aut,
cph] (<[https://orcid.org/0000-0002-2842-3406](https://url.uk.m.mimecastprotect.com/s/qh6YCl714TnNOvBUDTZFze4E_?domain=orcid.org)>), Flavio Finger
[aut] (<[https://orcid.org/0000-0002-8613-5170](https://url.uk.m.mimecastprotect.com/s/5mreCmy54uJN1rmtEUGFR9LKu?domain=orcid.org)>), Zhian N.
Kamvar [ctb] (<[https://orcid.org/0000-0003-1458-7108](https://url.uk.m.mimecastprotect.com/s/uJFbCnO14cONlYJf1cnFJJyTo?domain=orcid.org)>), Hugo
Gruson [ctb, rev] (<[https://orcid.org/0000-0002-4094-1476](https://url.uk.m.mimecastprotect.com/s/NKxUCoy14uq3l07Urf0Fp1Ip0?domain=orcid.org)>),
Karim Mané [rev] (<[https://orcid.org/0000-0002-9892-2999](https://url.uk.m.mimecastprotect.com/s/TMdYCp215iM3QBjiph1FGqvrA?domain=orcid.org)>),
Pratik Gupte [rev] (<[https://orcid.org/0000-0001-5294-7819](https://url.uk.m.mimecastprotect.com/s/nYexCqZ14t0jLopt1i9FEtDre?domain=orcid.org)>),
Joshua W. Lambert [rev]
(<[https://orcid.org/0000-0001-5218-3046](https://url.uk.m.mimecastprotect.com/s/8PY7Cr914SVlwG5sDsxF4wtoM?domain=orcid.org)>)
Maintainer: James M. Azam <james.azam@lshtm.ac.uk>
Depends: R (>= 3.6.0)
Suggests: bookdown, dplyr, epicontacts, ggplot2, knitr, lubridate,
rmarkdown, spelling, testthat, truncdist
Description: Provides methods to simulate and analyse the size and length
of branching processes with an arbitrary offspring
distribution. These can be used, for example, to analyse the
distribution of chain sizes or length of infectious disease
outbreaks, as discussed in Farrington et al. (2003)
<doi:10.1093/biostatistics/4.2.279>.
License: MIT + file LICENSE
Imports: checkmate, stats, utils


Submitter's comment: ## R CMD check results

0 errors | 0 warnings | 1
note

* This is a new release.

**Comment by jamesmbaazam** [comment]
Dear maintainer,

package epichains_0.1.1.tar.gz has been auto-processed and is pending a manual inspection of this new CRAN submission. A CRAN team member will typically respond to you within the next 10 working days. For technical reasons you may receive a second copy of this message when a team member triggers a new check.

Log dir: <[https://win-builder.r-project.org/incoming_pretest/epichains_0.1.1_20241009_135554/](https://url.uk.m.mimecastprotect.com/s/OgNBC3wNDcAzRQpHDsOFQO15D?domain=win-builder.r-project.org)>
The files will be removed after roughly 7 days.
Installation time in seconds: 4
Check time in seconds: 95
R Under development (unstable) (2024-10-08 r87214 ucrt)

Pretests results:
Windows: <[https://win-builder.r-project.org/incoming_pretest/epichains_0.1.1_20241009_135554/Windows/00check.log](https://url.uk.m.mimecastprotect.com/s/0qBUC4LO0C0K71BtVtEF42Bba?domain=win-builder.r-project.org)>
Status: 1 NOTE
Debian: <[https://win-builder.r-project.org/incoming_pretest/epichains_0.1.1_20241009_135554/Debian/00check.log](https://url.uk.m.mimecastprotect.com/s/56lnC5LPGCjLWGZi8u9Fkigu0?domain=win-builder.r-project.org)>
Status: 1 NOTE



No strong reverse dependencies to be checked.

Best regards,
CRAN teams' auto-check service

**Pull request merged**
